templates/app/Dockerfile

```
@@ -1,5 +1,16 @@
-FROM amazoncorretto:11
+FROM amazoncorretto:11 as builder
+RUN mkdir -p /opt/app
+WORKDIR /opt/app
+COPY build.gradle.kts .
+COPY gradle.properties .
+COPY gradle gradle/
+COPY gradlew .
+COPY src src/
+RUN ./gradlew build
```

```suggestion
RUN ./gradlew bootJar
```
Why run a longer task than necessary?

---

Do you think is OK don't execute klint? I tested and bootJar do not runs it.

---

I think ktlint should run in another moment, maybe a CI build.